### ISSUE TITLE
Does not work with Spring Boot 2.2.1 / Spring Data 2.2.1

### ISSUE THREAD

**Issue by patkovskyi** [derjust/spring-data-dynamodb#273]
## Expected Behavior
Works with Spring Boot 2.2.1 / Spring Data 2.2.1

## Actual Behavior
Does not work with Spring Boot 2.2.1 / Spring Data 2.2.1

## Steps to Reproduce the Problem

  1. Use Spring Boot 2.2.1 without specifying `implementation 'org.springframework.data:spring-data-commons:2.2.0.RELEASE'`.
  2. DynamoDB repository beans are not created (org.springframework.beans.factory.NoSuchBeanDefinitionException).

**Workaround (need to verify if everything is really ok)**:
Downgrade Spring Data to 2.2.0: `implementation 'org.springframework.data:spring-data-commons:2.2.0.RELEASE'`

## Specifications
Spring Data DynamoDB Version: 5.1.0 (2.1)
Spring Data Version:          2.2.1.RELEASE
AWS SDK Version:              1.11.443
Java Version:                 12.0.2 - OpenJDK 64-Bit Server VM 12.0.2+10
Platform Details:             Mac OS X 10.14.6

**Comment by boostchicken** [comment]
This is due to lack of multiple repository support probably. I have a fix incoming sometime soon.

**Comment by boostchicken** [comment]
@patkovskyi     
Fixed in 5.2.1-SNAPSHOT under my fork, if you could open an issue on my repo for tracking purposes I would appreciate it.  That is OSSRH snapshots, have a release soon.
```
repositories {
        mavenCentral()
        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
    }
```

`api  group: 'io.github.boostchicken', name: 'spring-data-dynamodb', version: '5.2.1-SNAPSHOT'
`
https://github.com/boostchicken/spring-data-dynamodb

**Comment by boostchicken** [comment]
api  group: 'io.github.boostchicken', name: 'spring-data-dynamodb', version: '5.2.1'

On maven central now

**Comment by boothen** [comment]
@boostchicken I was seeing this issue. Your new version has fixed it. Thanks for that.

Have you just volunteered to take over development and maintenance of the repo? 😉

**Comment by boostchicken** [comment]
@boothen 
Feel free to follow my repo, I will continue to support it. Contributions welcome :)

**Comment by tobiashochguertel** [comment]
Does this Fix come also get into this repository?

**Comment by btyoshidakim** [comment]
When will you release it as a new version of `spring-data-dynamodb`?

**Comment by gauravbrills** [comment]
> io.github.boostchicken

will this be merged to this repo is there a PR ?

**Comment by boostchicken** [comment]
@gauravbrills I would say no.  Move to my fork.

**Comment by gauravbrills** [comment]
> @gauravbrills I would say no. Move to my fork.

Yup doing the same for now but can't there be one standard repo or if spring cloud aws or data can merge this in the main stream. This has now shifted across 3 repos now.

@derjust any update if this can be merged
